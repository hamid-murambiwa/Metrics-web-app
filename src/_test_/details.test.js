import React from 'react';
import renderer from 'react-test-renderer';
import { Provider } from 'react-redux';
import { BrowserRouter as Router } from 'react-router-dom';
import store from '../redux/configureStore';
import Details from '../pages/details';

describe('Render components', () => {
  test('render HomePage', () => {
    const tree = renderer
      .create(<Router><Provider store={store}><Details /></Provider></Router>)
      .toJSON();
    expect(tree).toMatchSnapshot();
  });
});
