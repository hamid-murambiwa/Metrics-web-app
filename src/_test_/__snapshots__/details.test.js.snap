// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Render components render HomePage 1`] = `
<div>
  <div>
    <header>
      <a
        href="/"
        onClick={[Function]}
      >
        <img
          alt=""
          className="h-img"
          src="icons8-back-90.png"
        />
      </a>
      <div
        className="s-con"
      >
        <a
          href="/"
          onClick={[Function]}
        >
          <img
            alt=""
            className="h-img s"
            src="icons8-microphone-90.png"
          />
        </a>
        <a
          href="/"
          onClick={[Function]}
        >
          <img
            alt=""
            className="h-img"
            src="icons8-settings.svg"
          />
        </a>
      </div>
    </header>
  </div>
  <section
    id="d-con"
  >
    <div
      id="i-con"
    >
      <img
        alt=""
        id="breed-img"
      />
      <h1 />
    </div>
    <ul
      className="details"
    >
      <li>
        <h2>
          Breed group
        </h2>
        <p />
      </li>
      <li>
        <h2>
          Breed for
        </h2>
        <p />
      </li>
      <li>
        <h2>
          Country origin
        </h2>
        <p />
      </li>
      <li>
        <h2>
          Characteristics
        </h2>
        <p />
      </li>
      <li>
        <h2>
          Height
        </h2>
        <p />
        <p />
      </li>
      <li>
        <h2>
          Weight
        </h2>
        <p />
        <p />
      </li>
    </ul>
  </section>
</div>
`;
